mutatex/utils.py

```diff
@@ -208,6 +207,14 @@ def parse_poslist_file(fname):
             log.error("the position list file is not in the right format")
             log.error("format error at %s" % line.strip())
             raise TypeError
+        if residue not in unique_residues:
+            pdb_residues_list=[]
+            for i in unique_residues:
+                single_set=set(i)
+                pdb_residues_list.append(set(residue).issubset(single_set))
+            if True not in pdb_residues_list:
```

Comment:
you can also check if there is more than one `True` value (which is expected in case the residue specification is ambiguous, e.g. `SA35_TB36`

Reply:
i.e. 
```
if pdb_residues_list.count(True) != 1:
```

Reply:
done